modules/web/src/de/diedavids/cuba/dataimport/web/importattributemapper/import-attribute-mapper-edit.xml

```diff
@@ -22,7 +22,9 @@
                 <fieldGroup id="fieldGroup"
                             datasource="importAttributeMapperDs">
                     <column width="250px">
-                        <field property="entityAttribute"/>
+                        <field property="entityAttribute">
+                            <lookupField id="entityAttributeId" datasource="importAttributeMapperDs" property="entityAttribute"/>
```

Comment:
Could you explain why not using directly XML ?

Reply:
Because I think it is the standard way of doing it - I guess? I have actually not seen this solution. 

There are some drawbacks. E.g. I would imagine that with this solution certain features are not directly supported anymore. E.g. when it comes to read-only certain attributes through the security settings.

I would just go the default way as long as it is not required to do otherwise.